lib/components/user/monitored-trip/trip-summary.tsx

```diff
@@ -0,0 +1,50 @@
+import { FormattedMessage, FormattedTime } from 'react-intl'
+import React, { useContext } from 'react'
+
+import { ComponentContext } from '../../../util/contexts'
+import FormattedDuration from '../../util/formatted-duration'
+import InvisibleA11yLabel from '../../util/invisible-a11y-label'
+
+interface Props {
+  // TODO: use a more complete definition of monitored trip.
+  monitoredTrip: {
+    itinerary: {
+      duration: number
+      endTime: number
+      startTime: number
+    }
+  }
+}
+
+const TripSummary = ({ monitoredTrip }: Props): JSX.Element => {
+  const { itinerary } = monitoredTrip
+  const { duration, endTime, startTime } = itinerary
+  // @ts-expect-error TODO: add ModesAndRoutes to ItineraryBody attribute of ComponentContext
+  const { ItineraryBody, LegIcon } = useContext(ComponentContext)
+  const ModesAndRoutes = ItineraryBody.ModesAndRoutes
+  return (
+    <div
+      className="otp option default-itin"
+      style={{ borderTop: '0px', padding: '0px' }}
+    >
+      <div className="header">
```

Comment:
What is the issue here @miles-grant-ibigroup? Is this about the text highlight cursor showing up?

Reply:
I was expecting `cursor: text` on this element

Reply:
What is the purpose? The SVG images are not natively selectable.

Reply:
I was thinking about the text part

Reply:
The text should already have the correct (default) cursor:
![Screen Recording 2023-03-01 at 2 15 36 PM](https://user-images.githubusercontent.com/56846598/222242253-86dcc363-ee44-45d9-8623-acf29f69a7c6.gif)

Reply:
Sorry I should've been more clear! I was talking about the "please provide a name..."

Reply:
Ah okay. It looks like the default for labels is `pointer`. What you are seeing is that the default cursor for the notification options "label" (which is actually a `<legend>` within a `<fieldset>`) is `text`.

Reply:
I don't think it should be though right? Can we override it?

Reply:
Yes, per teams chat, this will be done in #799 (see https://github.com/opentripplanner/otp-react-redux/pull/799/commits/a0641594a362fd0f3d6f9aafe1e10ad1aa41c95d).